tests/deepsparse/transformers/pipelines/test_text_generation.py

```diff
@@ -104,6 +104,29 @@ def test_token_generation_non_deterministic(pipeline, prompt):
     assert len(set(text_outputs)) == 3
 
 
+def test_pipeline_for_ppl_eval(self):
```

Comment:
Yes, the test does not contemplate accuracy, only functionality. It assesses that the pieces needed from the pipeline are present, so changes to the pipeline don't break ppl evaluation. I don't think that assessing accuracy would be in the scope of a unit test.

I just realized that the testing interface changed and that the unit test should take a pipeline and a prompt instead of "self". I'm fixing that